src/util/matchJson.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Compares two JSON-like objects, returning false if they differ.
+ */
+
+export function matchJson(a: any, b: any): boolean {
+  // Use simple equality, unless a and b are proper objects:
+  if (
+    typeof a !== 'object' ||
+    typeof b !== 'object' ||
+    a == null ||
+    b == null
+  ) {
+    return a === b
+  }
+  // These must either be both arrays or both objects:
+  const aIsArray = Array.isArray(a)
+  const bIsArray = Array.isArray(b)
+  if (aIsArray !== bIsArray) return false
+  // Compare arrays in order:
+  if (aIsArray) {
+    if (a.length !== b.length) return false
+    for (let i = 0; i < a.length; ++i) {
+      if (!matchJson(a[i], b[i])) return false
+    }
+    return true
+  }
+  // These are both regular objects, so grab the keys:
+  const aKeys = Object.getOwnPropertyNames(a)
```

Comment:
This is the old corrupted copy from the GUI. Please grab the fresh version from edge-server-tools.

Reply:
Done. Updated w/fixup commit